src/modules/Genre/Genre.js

```diff
@@ -119,8 +120,11 @@ function Genre() {
             </div>
           </div>
         </div>
-        <div className="song-name col">
+        <div className="song-name list-col">
           {track.trackIndex + '. ' + track.songName}
+          <pre>
+            {"Artist: " + track.artistName}
```

Comment:
Any reason why the artist is in a different font?

Reply:
Also, not sure if you need the "Artist: " text. The wireframe doesn't have that: 
![image](https://user-images.githubusercontent.com/49804462/128416324-0dfe67c5-d6a8-47e2-99aa-31cba662017f.png)

Reply:
Also, I think the artist text should be aligned vertically with the song name, similar to the wireframe and what I've done: 
![image](https://user-images.githubusercontent.com/49804462/128416700-c0cbc766-e70c-42d3-9af4-4d43b42a20e8.png)
Looks better in my opinion.

Reply:
@lchua2314 
I needed a way to separate the artist and song name.
"\n" didn't work so I used a new font style for artist so that it could separate properly

Reply:
> @leonzh2k
> I needed a way to separate the artist and song name.
> "\n" didn't work so I used a new font style for artist so that it could separate properly

Which one of my comments is this referencing?

Reply:
OK, I know which comment you're responding to. Can't you use a `<div>` or a `<span>` instead?

Reply:
> @leonzh2k
> I needed a way to separate the artist and song name.
> "\n" didn't work so I used a new font style for artist so that it could separate properly

OK, I know which comment you're responding to. Can't you use a `<div>` or a `<span>` instead?

Reply:
> > @leonzh2k
> > I needed a way to separate the artist and song name.
> > "\n" didn't work so I used a new font style for artist so that it could separate properly
> 
> OK, I know which comment you're responding to. Can't you use a `<div>` or a `<span>` instead? Those won't change the font.
alright I fixed the artist spacing alignment and used <div> to maintain the same font type.
It looks like this now
![image](https://user-images.githubusercontent.com/30790329/128418813-fbd705e8-7314-437e-bc09-7098d0362afc.png)

![image](https://user-images.githubusercontent.com/30790329/128418637-1f48db94-b749-4136-a7fb-b830950a87bf.png)

Reply:
alright I fixed the artist spacing alignment and used <div> to maintain the same font type.
It looks like this now
![image](https://user-images.githubusercontent.com/30790329/128418813-fbd705e8-7314-437e-bc09-7098d0362afc.png)

![image](https://user-images.githubusercontent.com/30790329/128418637-1f48db94-b749-4136-a7fb-b830950a87bf.png)

Reply:
Cool, looks good now.